input/vulnerabilities/Browser_Cross-App_Scripting.json

```diff
@@ -1,7 +1,7 @@
 {
   "name" : "Browser Cross-App Scripting",
   "CVE" : [["CVE-2011-2357","watchfire-crossapp"]],
-  "Responsibly_disclosed" : true,
+  "Coordinated_disclosure" : "true",
   "Categories" : ["app"],
   "Details" : [["Android browser could be tricked into running javascript in the domain of a different app","watchfire-crossapp"]],
   "Severity" : "uncertain",
```

Comment:
The "Severity" field probably needs deprecating and potentially removing.

Reply:
Will add a note to the documentation that it's deprecated